app/src/main/java/de/xikolo/config/Feature.kt

```diff
@@ -20,6 +20,10 @@ object Feature {
         Build.VERSION.SDK_INT >= 26 && App.instance.packageManager
             .hasSystemFeature(PackageManager.FEATURE_PICTURE_IN_PICTURE)
 
+    @JvmField
```

Comment:
Is that flag actually be used by any Java code? If not, remove the annotation and make it a `const`.

Reply:
Can you check if I fixed the const correctly? Not sure if that's what you've meant

Reply:
Yup, fine.